CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.8)
-project(checkpoint)
+project(checkpoint VERSION 1.1.0)
```

Comment:
If there are no merges with checkpoint during a given week, are we still going to bump the version to keep it in sync with vt? Given the various packages are currently falling under the 1.0.0 toolkit, I don't think it's critical that the checkpoint version match the vt version. Given that, is 1.1.0 the right place to start with checkpoint? I'm okay either way, but think we should probably put a little thought into this if we haven't already.

Reply:
There was a verbal agreement to keep it in sync with vt at the last DARMA weekly, but without much discussion.
No strong opinion here either.

Another thing to note:
- we currently test `vt` with `checkpoint`@`develop` (and with `detector`@`master`): 
https://github.com/DARMA-tasking/vt/blob/525c7d0ef5ad703629b55fdf2dcd91e2eab7956c/ci/build_cpp.sh#L10
- should `vt` releases point to a specific versions of the libs?

Reply:
`checkpoint` version should be aligned with `vt` - confirmed on DARMA weekly.